scipy/optimize/_linprog_util.py

```diff
@@ -334,10 +365,10 @@ def _clean_inputs(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None, bounds=None):
             "Invalid input for linprog: bounds must be a sequence of "
             "(min,max) pairs, each defining bounds on an element of x ")
 
-    return c, A_ub, b_ub, A_eq, b_eq, bounds
+    return c, A_ub, b_ub, A_eq, b_eq, bounds, x0
```

Comment:
We're often passing  ``c, A_ub, b_ub, A_eq, b_eq, bounds, x0`` between functions. Maybe this should be refactored into a container e.g. a namedtuple or dictionary?

Reply:
Deferred to #9671